non_atomic_component_message_processing/section_subject_domain_of_dialog_system_messages/subject_domain_of_dialog_system_message_understanding/svo_triple/rrel_svo_triple_element_values.scs

```diff
@@ -0,0 +1,66 @@
+rrel_svo_triple_element_values
+<- sc_node_role_relation;
+<- relation;
+<- role_relation;
+<- binary_relation;
```

Comment:
looks like a quasybinnary relation
should it have plural in all idtfs and definitions?